src/main/java/thePackmaster/cards/basicspack/SimplifiedStrike.java

```diff
@@ -0,0 +1,59 @@
+package thePackmaster.cards.basicspack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SimplifiedStrike extends AbstractPackmasterCard {
+    public final static String ID = makeID("SimplifiedStrike");
+
+    public SimplifiedStrike() {
+        super(ID, 2, CardType.ATTACK, CardRarity.RARE, CardTarget.ENEMY, "basics");
+        this.tags.add(CardTags.STRIKE);
+        this.baseMagicNumber = this.magicNumber = 3;
+        this.baseDamage = this.damage = 0;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        int basicCount = 0;
+        for (AbstractCard c : AbstractDungeon.actionManager.cardsPlayedThisCombat) {
+            if (c.rarity == CardRarity.BASIC)
+                basicCount++;
+        }
+        this.baseDamage = this.magicNumber * basicCount;
```

Comment:
Kinda jank. You could just calculate it in calculateCardDamage correctly so you didn't have to do this.